environment.yml

```diff
@@ -0,0 +1,12 @@
+name: pyds-env
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.9
```

Comment:
`3.8` doesn't work?

Reply:
In python 3.8 version the pytest is not working properly on the test cases present in the project.